File: pascal_triangle/PascalTriangle.scala
object PascalTriangle {
  def main(args: Array[String]): Unit = {
    println("Enter no. of rows")
    val row = scala.util.Try(readInt())
    if(row.isSuccess && row.get > 0){
      val pascalSum = scala.math.pow(2,row.get) - 1
      println(s"Pascal Triangle Sum = ${pascalSum}")
    }else{
      println("Enter valid row")
    }
  }
}

File: pascal_triangle/kotlin/src/PascalTriangle.kt
import java.util.*

object PascalTriangle {
    @JvmStatic fun main(args: Array<String>) {
        println("Enter row number: ")
        val rowNum = Scanner(System.`in`).nextInt()
        print(Math.pow(2.0, rowNum.toDouble()))
    }
}